GiEnJul/Controllers/AdminController.cs

```diff
@@ -37,8 +37,8 @@ public AdminController(IEventRepository eventRepository, IGiverRepository giverR
         // }
         
         [HttpGet("givers")]
-        public async Task<List<Models.Giver>> GetGiversAsync() {
-            return _mapper.Map<List<Models.Giver>>(await _giverRepository.GetAllAsync());
+        public async Task<List<Giver>> GetGiversAsync() {
+            return _mapper.Map<List<Giver>>(await _giverRepository.GetAllAsModelAsync());
```

Comment:
Is the mapping to List<Giver> necessary? If I understand correctly, the GetGiversAsync() function could return IEnumerable instead of List, and the mapping could be removed.

Reply:
You got it right! Need to fix that